### PR TITLE
Modified README Contributing section

### PR DESCRIPTION
**Pull request by konifar**
## Issue
- none

## Overview (Required)
I modified the Contributing section to make the contribution easier.
Thanks for a_akira 's suggestion!
https://twitter.com/_a_akira/status/952040204004507648

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: README.md [hunk 1]
```diff
@@ -9,11 +9,21 @@
 TBD
 
 # Contributing
-We use [waffle.io](https://waffle.io/DroidKaigi/conference-app-2018) to manage tasks. If you'd like to contribute to the project but are not sure where to start off, please look for issues labelled [welcome contribute](https://github.com/DroidKaigi/conference-app-2018/labels/welcome%20contribute).
+We are always welcome your contribution!
```
Comment (takahirom) [inline README.md]: 🆒

**Comment by takahirom** [comment]
It became quite easy to understand 💯

**Comment by takahirom** [comment]
@AAkira What do you think?

**Comment by takahirom** [comment]
OK
 https://twitter.com/_a_akira/status/952067893000220672
I will merge this!

**Comment by konifar** [comment]
Thanks @AAkira and @takahirom !

**Pull request merged**